js/analytics.js

```diff
@@ -5,15 +5,26 @@
 var URI = require('urijs');
 var _ = require('underscore');
 
+function trackerExists() {
+  if (typeof ga !== 'undefined') {
+    return true;
+  }
+}
```

Comment:
For this function, which I didn't explain very well, the idea was that `ga` is not used by any other code, it would only be used within `analytics.js`. `analytics` could expose a `track` function, would check if `ga` exists and then call `ga`. If it's not available, then `track` is a no-op. This means that external code always calls `analytics.track` and never has to check whether `ga` exists or not, it lets `track` decide what to do.